File: modules/scala/futureOpenTracing/src/main/java/io/opentelemetry/javaagent/instrumentation/scalafuture/VirtualFields.java
package io.opentelemetry.javaagent.instrumentation.scalafuture;

import io.opentelemetry.javaagent.shaded.instrumentation.api.util.VirtualField;
import io.opentelemetry.javaagent.bootstrap.executors.PropagatedContext;

public final class VirtualFields {

    public static final VirtualField<Runnable, PropagatedContext> RUNNABLE_PROPAGATED_CONTEXT =
            VirtualField.find(Runnable.class, PropagatedContext.class);

    private VirtualFields() {}
}

File: modules/scala/futureOpenTracing/src/main/scala/io/opentelemetry/javaagent/instrumentation/scalafuture/VirtualFields.scala
package io.opentelemetry.javaagent.instrumentation.scalafuture

import io.opentelemetry.javaagent.bootstrap.executors.PropagatedContext
import io.opentelemetry.javaagent.shaded.instrumentation.api.util.VirtualField

object VirtualFields {

  val RUNNABLE_PROPAGATED_CONTEXT: VirtualField[Runnable, PropagatedContext] =
    VirtualField.find(classOf[Runnable], classOf[PropagatedContext])
}
